src/debugpy/_vendored/pydevd/pydevd_attach_to_process/linux_and_mac/compile_mac.sh

```diff
@@ -1,4 +1,14 @@
-set -e
-SRC="$(dirname "$0")/.."
-g++ -fPIC -D_REENTRANT -std=c++11 -D_FORTIFY_SOURCE=2 -arch x86_64 -c $SRC/linux_and_mac/attach.cpp -o $SRC/attach_x86_64.o
-g++ -dynamiclib -nostartfiles -arch x86_64 -lc $SRC/attach_x86_64.o -o $SRC/attach_x86_64.dylib
+#!/bin/bash
+set -euxo pipefail
+
+clang++ -fPIC -D_REENTRANT -std=c++11 -arch arm64 -c -o attach_arm64.o attach.cpp
```

Comment:
I assume clang supports D_FORTIFY_SOURCE?. We have internal requirements on this flag being set or Microsoft security won't let us ship this binary. Can you try adding it back?

Reply:
And why use clang instead of g++?

Reply:
The build script was pulled from https://github.com/JetBrains/intellij-community/commit/2e7109f6fe37cf3ba74778b72bfbe9ddeb8ca4b4. I've pushed up a new version that is more like the original.